vision/tests/system_old.py

```diff
@@ -200,9 +200,9 @@ def _assert_logo(self, logo):
         self.assertEqual(logo.bounds.vertices[1].x_coordinate, 940)
         self.assertEqual(logo.bounds.vertices[1].y_coordinate, 40)
         self.assertEqual(logo.bounds.vertices[2].x_coordinate, 940)
-        self.assertEqual(logo.bounds.vertices[2].y_coordinate, 302)
+        self.assertEqual(logo.bounds.vertices[2].y_coordinate, 243)
         self.assertEqual(logo.bounds.vertices[3].x_coordinate, 40)
-        self.assertEqual(logo.bounds.vertices[3].y_coordinate, 302)
+        self.assertEqual(logo.bounds.vertices[3].y_coordinate, 243)
```

Comment:
This is a perfect example of why assertions about "magic" values returned by the back end are a bad idea: they can break without any chance of a fix (and probably will again, sooner or later).